Running-Mastodon/Docker-Guide.md

```diff
@@ -14,10 +14,9 @@ The project now includes a `Dockerfile` and a `docker-compose.yml` file (which r
 
 Clone Mastodon's repository.
     
-    # Clone mastodon to ~/live directory
+    # Clone mastodon to live directory
     git clone https://github.com/tootsuite/mastodon.git live
```

Comment:
Why not just change live to ~/live here?

Reply:
You are thinking about something like this:
```diff
-git clone https://github.com/tootsuite/mastodon.git live
+ git clone https://github.com/tootsuite/mastodon.git ~/live
```

I would prefer not to clone to home but rather to the current working directory. If you want to clone into home then just switch to it prior to calling the git clone command.

The current changes would make it easier to use, in my opinion. No need for a fixed directory here.

Reply:
I mean, the guide makes you `su - mastodon` before. You're storing the mastodon code in $HOME of mastodon user. We use `live` folder name (although I don't like it) because having it be `/home/mastodon/mastodon` might be quite confusing to people.

Reply:
> having it be /home/mastodon/mastodon might be quite confusing to people.

I totally agree,
#### but
We are probably talking about different guides here. Maybe therefore the confusion.

I edited the docker documentation.
For the docker installation there is no need to change the current user.